index.ts

```diff
@@ -1359,6 +1520,7 @@ export const translations = {
         "Glijden",
         "Klouzavost",
         "Planare",
+        "Ślizg",
```

Comment:
glide - In what meaning is that? As a noun (type of plain), a verb etc.?

Reply:
This term is used to distinct an flight phase (opposite of thermaling).